platforms/android/example-compose/src/main/java/io/element/wysiwyg/compose/MainActivity.kt

```diff
@@ -38,13 +40,25 @@ class MainActivity : ComponentActivity() {
                             onResetText = {
                                 state.setHtml("")
                             },
-                            onBoldClick = {
-                                state.toggleBold()
-                            },
-                            onItalicClick = {
-                                state.toggleItalic()
-                            },
                             actionStates = state.actions,
+                            onActionClick = { action ->
+                                when(action) {
+                                    ComposerAction.BOLD -> state.toggleInlineFormat(InlineFormat.Bold)
```

Comment:
This looks like boilerplate. If users of the library must always handle all actions, shouldn't we put this logic in the state holder and just expose `state.handleAction()` or something?